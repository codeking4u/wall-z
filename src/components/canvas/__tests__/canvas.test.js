import renderer from 'react-test-renderer'

import Canvas from '../canvas.component'

test('matches snapshot', () => {
    const tree = renderer.create(<Canvas />).toJSON();
    expect(tree).toMatchSnapshot();
})