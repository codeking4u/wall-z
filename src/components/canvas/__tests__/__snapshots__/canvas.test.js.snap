// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<canvas
  className="canvas-pg undefined"
  height={0}
  onClick={[Function]}
  onMouseMove={[Function]}
  style={
    Object {
      "backgroundImage": "url(Sample_Floorplan.jpg)",
      "backgroundPosition": "center",
      "backgroundRepeat": "no-repeat",
      "backgroundSize": "contain",
    }
  }
  width={0}
/>
`;
